### DISCUSSION TITLE
Default transition time for all devices?

### DISCUSSION THREAD

**Discussion by lougreenwood** [Koenkk/zigbee2mqtt#17024] [General]
Hello, I'm testing out Z2M after having some issues with ZHA, Sonoff Plus V2 (aka `E`) dongle & Philips Hue SML001 sensors...

I have a question, is it possible to set a default, system-wide transition time for all lights? ZHA has this on the main config page, but I can't find anything like this for Z2M in the docs or generally on google.

Thanks!

**Comment by lux73** [comment]
configuration.yaml

```
device_options:
  transition: 0.4
```

**Reply by lux73** [reply]
btw Sonoff Dongle-E is not recommended - very buggy Firmware afaik

**Reply by lougreenwood** [reply]
Thanks - so far it's working fine for the past few days. I received a `P` today too, but not sure I'll bother swapping them out.

**Edit:**
Only issue I had so far was pairing Philips SML001 motion sensors, most of the time they failed pairing / interview and needed multiple attempts.

**Reply by lux73** [reply]
search around the Forums - if you want a strong & reliable Zigbee Network i would swap them. The Dongle-P is so much better in my opinion ;-)
